CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+
+The first thing to figure out is exactly what you’re going to contribute!
+We describe all future work as individual [github issues](https://github.com/software-gardening/almanac/issues).
+For first time contributors we have specifically labeled as [good first issue](https://github.com/software-gardening/almanac/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
+
+If you want to contribute code that we haven’t already outlined, please start a discussion in a new issue before writing any code.
+A discussion will clarify the new code and reduce merge time.
+Plus, it’s possible your contribution belongs in a different code base, and we do not want to waste your time (or ours)!
+
+### Pull requests
+
+After you’ve decided to contribute code and have written it up, please file a pull request.
+We specifically follow a [forked pull request model](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).
+Please create a fork of Software Gardening Almanac repository, clone the fork, and then create a new, feature-specific branch.
+Once you make the necessary changes on this branch, you should file a pull request to incorporate your changes into the main The Software Gardening Almanac repository.
+
+The content and description of your pull request are directly related to the speed at which we are able to review, approve, and merge your contribution into The Software Gardening Almanac.
+To ensure an efficient review process please perform the following steps:
+
+1. Follow all instructions in the [pull request template](https://github.com/software-gardening/almanac/blob/main/.github/PULL_REQUEST_TEMPLATE.md)
+1. Triple check that your pull request is adding _one_ specific feature. Small, bite-sized pull requests move so much faster than large pull requests.
+1. After submitting your pull request, ensure that your contribution passes all status checks (e.g. passes all tests)
+
+Pull request review and approval is required by at least one project maintainer to merge.
+We will do our best to review the code additions in a timely fashion.
+Ensuring that you follow all steps above will increase our speed and ability to review.
+We will check for accuracy, style, code coverage, and scope.
+
+### Git commit messages
+
+For all commit messages, please use a short phrase that describes the specific change.
+For example, “Add feature to check normalization method string” is much preferred to “change code”.
+When appropriate, reference issues (via `#` plus number) .
+
+## Development
+
+### Overview
+
+The Software Gardening Almanac is primarily written in [Python](https://www.python.org/) through [Jupyter Book](https://jupyterbook.org/) with related environments managed by Python [Poetry](https://python-poetry.org/).
+[Node](https://nodejs.org) and [NPM](https://www.npmjs.com/) dependencies are also used to assist development activities.
```

Comment:
```suggestion
We write and develop the Software Gardening Almanac in [Python](https://www.python.org/) through [Jupyter Book](https://jupyterbook.org/) with related environments managed by Python [Poetry](https://python-poetry.org/).
We use [Node](https://nodejs.org) and [NPM](https://www.npmjs.com/) dependencies to assist development activities.
```

To move to active voice. Also, it might be worth clarifying (or providing an example) of the development activities.

Reply:
Thanks, good point! I'll add this information in with more content.